bookstore-nestjs/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:lts-alpine AS BASEIMAGE
+WORKDIR /src
+COPY package*.json ./
+RUN npm ci
+COPY . .
```

Comment:
I think it's better to be explicit in what we copy in a Dockerfile and only include the files that are needed

Reply:
So I'm not super familiar with Docker, which is why I was looking for some examples online for Dockerfiles with Nest. Pretty much all of the recent examples that I've found all used that copy command. 
Could you please suggest a solution because I'm not sure how to be explicit here? :)

https://www.tomray.dev/nestjs-docker-production
https://github.com/Saluki/nestjs-template/blob/master/Dockerfile
https://dev.to/erezhod/setting-up-a-nestjs-project-with-docker-for-back-end-development-30lg